internal/runtime/hcsv2/nvidia_utils.go

```diff
@@ -0,0 +1,77 @@
+// +build linux
+
+package hcsv2
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/Microsoft/opengcs/internal/storage/pci"
+	oci "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+)
+
+const lcowNvidiaMountPath = "/run/nvidia"

```

Comment:
I would add a comment here that this needs to match the path that is mapped to by the shim.

Reply:
Will do. Do you know of a better way to keep these two things synced?

Reply:
We could put this value in a package that is used in both places, but that seems like overkill to me.

Reply:
Yeah I agree. We'll keep it this way then for now.